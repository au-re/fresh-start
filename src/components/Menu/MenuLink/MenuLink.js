import { Link } from "react-router-dom";
import styled from "styled-components";

const MenuLink = styled(Link) `
  color: white;
  font-size: 1rem;
  display: block;

  :hover {
    text-decoration: none;
    color: white;
  }
`;

export default MenuLink;
