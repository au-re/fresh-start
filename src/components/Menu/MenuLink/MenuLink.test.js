import "jest-styled-components";

import React from "react";
import { MemoryRouter } from "react-router";
import renderer from "react-test-renderer";

import MenuLink from "./MenuLink";

describe("MenuLink", () => {

  it("renders without props", () => {
    const tree = renderer.create(
      <MemoryRouter>
        <MenuLink to="#"/>
      </MemoryRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
