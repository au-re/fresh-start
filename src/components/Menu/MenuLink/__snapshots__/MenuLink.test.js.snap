// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuLink renders without props 1`] = `
.c0 {
  color: white;
  font-size: 1rem;
  display: block;
}

.c0:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: white;
}

<a
  className="c0"
  href="#"
  onClick={[Function]}
/>
`;
